[program:comms_interface]
command=/home/cloud/env/bin/python /home/cloud/peoplefinder/com_interface/comms_interface.py
autostart=true
autorestart=true
stderr_logfile=/var/log/peoplefinder/comms_interface.err.log
stdout_logfile=/var/log/peoplefinder/comms_interface.out.log
stdout_logfile_maxbytes=10MB
stdout_logfile_backups=10
killasgroup=true
stopasgroup=true
environment=PYTHONPATH="${PYTHONPATH}:/home/cloud/peoplefinder"

[program:comms_interface_emulation]
command=/home/cloud/env/bin/python /home/cloud/peoplefinder/com_interface/comms_interface.py -t
autostart=false
autorestart=false
stderr_logfile=/var/log/peoplefinder/comms_interface_emulation.err.log
stdout_logfile=/var/log/peoplefinder/comms_interface_emulation.out.log
stdout_logfile_maxbytes=10MB
stdout_logfile_backups=10
killasgroup=true
stopasgroup=true
environment=PYTHONPATH="${PYTHONPATH}:/home/cloud/peoplefinder"
